src/wp-admin/options-general.php

```diff
@@ -190,16 +204,20 @@ class="<?php echo esc_attr( $classes_for_button ); ?>"
 		<button
 			id="js-remove-site-icon"
 			type="button"
-			<?php echo has_site_icon() ? 'class="button button-secondary reset"' : 'class="button button-secondary reset hidden"'; ?>
+			<?php echo has_site_icon() ? 'class="button button-secondary reset remove-site-icon"' : 'class="button button-secondary reset hidden"'; ?>
 		>
 			<?php _e( 'Remove Site Icon' ); ?>
 		</button>
 	</div>
 
 	<p class="description">
 		<?php
-			/* translators: %s: Site Icon size in pixels. */
-			printf( __( 'The Site Icon is what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. It should be square and at least %s pixels.' ), '<code>512 &times; 512</code>' );
+			/* translators: 1: opening  <code> tag. 2: closing </code> tag. */
+			printf(
+				__( 'The Site Icon is what you see in browser tabs, bookmark bars, and within the WordPress mobile apps. It should be square and at least %1$s512 by 512%2$s pixels.' ),
```

Comment:
the `512  &times; 512` was intentionally not translated since those values aren't changeable.  I think we should keep them as passed values.

Reply:
Thank you for the testing, I've addressed this concern in my latest commit. => [68e0a5b](https://github.com/WordPress/wordpress-develop/pull/6871/commits/68e0a5b7e237768ef05ddec98df7fc4528da201a)

Reply:
> the `512 &times; 512` was intentionally not translated since those values aren't changeable. I think we should keep them as passed values.

According to a [random page I found on the internet](https://www.omniglot.com/language/numerals.htm) (so it must be true 🙂) there are different numerals in use for non-latin languages. It would be good to get some input from some i18n experts to find out if they string ought to be translatable. cc @swissspidy @ocean90

Reply:
Maybe a case for the classic `number_format_i18n` which have been around since 2.3?

Reply:
There's precedence in core for not translating this, so let's keep the existing text at is.

But **please** do not use placeholders for HTML tags! Undo that change.

Reply:
Ok @swissspidy. New changes committed.